src/style.css

```diff
@@ -0,0 +1,75 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+html {
+  font-family: Avenir, serif;
+  font-weight: 300;
+}
+
+body {
+  margin: 20px 8px;
+}
+
+.buttonBig {
+  width: 19px;
+  position: absolute;
+  padding: 25px 26px;
+  cursor: pointer;
+}
+
+.buttonSmall {
+  width: 11px;
+  position: absolute;
+  padding: 15px 13px;
+  cursor: pointer;
```

Comment:
Don't repeat the same styles. Check the checklist, please. Fix it everywhere 
<img width="1674" alt="image" src="https://user-images.githubusercontent.com/99403198/219343901-1303eba3-b622-40ab-94db-299c2171ab9c.png">